### PR TITLE
GITOPSRVCE-750: Add metric

### PR DESCRIPTION
**Pull request by keithchong**
Apparently, the metric that I need is not getting forwarded from the rhtap clusters to rhobs

I chatted with @yftacherzog about this and he's been working with me regarding my alert rules too.

Jira Ticket
https://issues.redhat.com/browse/GITOPSRVCE-750

### CODE REVIEW

**Comment by jgwest** [comment]
```

Summarizing 3 Failures:
  [FAIL] [rhtap-demo-suite] DEVHAS-234: create an application with quarkus component [It] waits for quarkus-devfile component (private: false) pipeline to be finished [rhtap-demo]
  /tmp/tmp.MHmLAkBCV0/tests/rhtap-demo/rhtap-demo.go:239
  [FAIL] [rhtap-demo-suite] DEVHAS-234: create an nodejs application without dockerfile [It] waits for nodejs-no-dockerfile component (private: false) pipeline to be finished [rhtap-demo]
  /tmp/tmp.MHmLAkBCV0/tests/rhtap-demo/rhtap-demo.go:239
  [FAIL] [rhtap-demo-suite] DEVHAS-234: create an application with branch and context dir [It] waits for component-devfile-java-sample component (private: false) pipeline to be finished [rhtap-demo]
  /tmp/tmp.MHmLAkBCV0/tests/rhtap-demo/rhtap-demo.go:239
Ran 492 of 596 Specs in 1459.111 seconds
FAIL! -- 489 Passed | 3 Failed | 7 Pending | 97 Skipped
```

/retest

**Comment by yftacherzog** [comment]
@avi-biton @amisstea can you have a look? I want another set of eyes here to think whether having this in RHOBS will affect our control plane alerts. I think we might have had a couple of tickets around this, but some of it was left for later.

**Comment by yftacherzog** [comment]
@keithchong the PR is good as far as I can tell. We have an issue though. We have exiting alerts based on this metric but used for control plane argoCD instances. Once we start forwarding the metric from the data plane cluster, those alerts are going to be triggered and going to make some noise for quite some people.

All in all, we need to refine the existing alerts before we can merge this one. I hope we can make it happen in the next few days.

cc @amisstea

**Comment by avi-biton** [comment]
@yftacherzog So, do we want to first handle [RHTAPWATCH-514](https://issues.redhat.com/browse/RHTAPWATCH-514) and only then address this PR?

**Comment by yftacherzog** [comment]
> @yftacherzog So, do we want to first handle [RHTAPWATCH-514](https://issues.redhat.com/browse/RHTAPWATCH-514) and only then address this PR?

Yes. I believe we should.

**Review by yftacherzog — APPROVED** [review]
/lgtm

**Review by avi-biton — APPROVED** [review]
/lgtm
/approve

**Comment by keithchong** [comment]
Hi @yftacherzog , do you know why the tests are failing?  I noticed some of the other PRs have the similar problem.

**Comment by yftacherzog** [comment]
> Hi @yftacherzog , do you know why the tests are failing? I noticed some of the other PRs have the similar problem.

Don't know. infra-deployments can be very flaky at times. I don't think this should be related to your changes.

**Comment by keithchong** [comment]
> Hi @yftacherzog , do you know why the tests are failing? I noticed some of the other PRs have the similar problem.

It's still failing. Is there a team/person we can contact who can help us?  It's been failing for some time now.

**Comment by yftacherzog** [comment]
> > Hi @yftacherzog , do you know why the tests are failing? I noticed some of the other PRs have the similar problem.
> 
> It's still failing. Is there a team/person we can contact who can help us? It's been failing for some time now.

I think asking on the rhtap developers channel in Slack or looking into the owners file at the root of the repo should help.

**Pull request merged**
